panacea/src/pml_parser.yrl

```diff
@@ -127,10 +134,25 @@ action_attribute ->
 
 requires_expr ->
     drug '{' string '}'            : construct('$1', [], value_of('$3')).
-
+requires_expr ->
+    time '{' time_expr_list '}'    : construct('$1', [], value_of_time('$3')).
 requires_expr ->
     expression : '$1'.
 
+time_expr ->
```

Comment:
Do you think this should return a `construct` instead of just an integer?

I'm thinking about taking an AST and producing PML again, with this approach I think we lose everything inside a `time` block.

Something like:

`construct('$1', [{minutes, get_time('$3')}), validate_times('$3'))`

Which would produce: `{:time, [minutes: N], [list of inner times]}`